packages/e2e-test-utils/src/inserter.js

```diff
@@ -3,6 +3,11 @@
  */
 import { pressKeyWithModifier } from './press-key-with-modifier';
 
+// This selector is written to support the current and old inserter markup
+// because the performance tests need to be able to run across versions.
+const INSERTER_SEARCH_SELECTOR =
```

Comment:
This ensures the tests run on wp/5.4 branch too. We need to support two major releases for the performance tests in order to produce the recap on each plugin release post.